hazelcast-client/src/test/java/com/hazelcast/client/map/querycache/ClientQueryCacheBasicTest.java

```diff
@@ -96,7 +96,7 @@ public void setup() {
         if (useNearCache) {
             clientConfig.addNearCacheConfig(new NearCacheConfig()
                     .setName(TEST_MAP_NAME)
-                    .setCacheLocalEntries(true));
+                    .setInvalidateOnChange(true));
```

Comment:
Indeed, `cacheLocalEntries` option does not make sense for a client, EE side validation should also fail. Created an issue for that https://github.com/hazelcast/hazelcast-enterprise/issues/1183